apps/meteor/app/livechat/server/api/v1/contact.ts

```diff
@@ -47,19 +47,39 @@ API.v1.addRoute(
 			check(this.queryParams, {
 				email: Match.Maybe(String),
 				phone: Match.Maybe(String),
+				custom: Match.Maybe(String),
 			});
+			const { email, phone, custom } = this.queryParams;
 
-			const { email, phone } = this.queryParams;
-
-			if (!email && !phone) {
+			if (!email && !phone && !custom) {
 				throw new Meteor.Error('error-invalid-params');
 			}
+			let foundCF = {};
+			if (custom) {
+				const customObj = Object.fromEntries(
+					Array.from(new URLSearchParams(custom)).filter(([k, v]) => typeof k === 'string' && typeof v === 'string'),
```

Comment:
What is the intention of this filter? 👀

Reply:
guarantee that the user can only input regular strings, not objects or other crazy things

Reply:
you're right, i could not get an array or object to pass through.